README.md

```diff
@@ -24,7 +24,7 @@ sudo pip install virtualenv
 virtualenv venv
 source venv/bin/activate
 pip install -r requirements.pip
-make html
+bash html
```

Comment:
What if I want to use my secret target that removes .pyc and .tmp files? `bash souji` ?